### ISSUE TITLE
[PROPOSAL] Extend documentation

### ISSUE THREAD

**Issue by gomorizsolt** [urish/firebase-server#139]
There's #79 which is similar to the description below.

Our team is currently using **`firebase-server`** as a reliable backend service for testing purposes. However, there was an obstacle in connection with the setup flow - this might have happened due to the lack of our experience.

We ended up with the basic concept below:
- the server should be started *before* the tests and also be stopped gracefully *after* all of them are done.

As a result, our proposal is to extend the documentation with a practical example that demonstrates how to setup the server with the usage of global test hooks.

**Example**

Let's suppose that the main testing framework is *Mocha*.

```js
// => e.g. global-test-hooks.js
const FirebaseServer = require("firebase-server");

let firebaseServer;

before(() => {
   firebaseServer = new FirebaseServer(5000, "localhost");
});

after(async () => {
   await firebaseServer.close();
});
```

```
// => require the file with the --file flag
mocha --file /path/to/global-test-hooks.js
```

**TL;DR**

It'd be useful to mention in the doc the way of setting up the server with the global test hooks - regardless of the applied framework.

All kinds of feedbacks and further proposals are welcome, even if it's completely disagreed.

**Comment by urish** [comment]
Thanks for the feedback @gomorizsolt , sounds like a useful addition! Can you send a pull-request adding the suggested example?

**Comment by gomorizsolt** [comment]
@urish of course, sorry for the late response. Check #140. ;)
